src/util.js

```diff
@@ -42,7 +58,9 @@ function shouldUpgrade( service, newInfo ) {
     //log("in shouldUpgrade in util.js: current's owner = " + info.owner + " new images owner = " + newInfo.owner);
     //log("in shouldUpgrade in util.js: current's repository = " + info.repository + " new images repository = " + newInfo.repository);
     //log("in shouldUpgrade in util.js: current's branch = " + info.branch + " new images branch = " + newInfo.branch);
-	var compatible = info.owner === newInfo.owner &&
+	var vailid = (info.owner !== "") && (info.repository !== "") && (info.branch !== "") && (info.version !== "") &&  (info.build !== "") &&  (info.commit !== "");
```

Comment:
@EvanSnapp - minor typo and indentation issue with this line. Just unindent one tab and change `vailid` to `valid` :+1: